packages/admin-app/src/components/GuardianIcon/GuardianIcon.tsx

```diff
@@ -38,7 +38,9 @@ export interface GuardianIconProps {
  * A menu option card for the menu screens
  */
 export const GuardianIcon: React.FC<GuardianIconProps> = ({ color, sequenceOrder, missing }) => {
-    const classes = useStyles(color, missing);
+    const theme = useTheme();
+    const colorOrDefault = color || theme.palette.primary.main;
```

Comment:
This is sad. I miss the colored guardians. This being said. This could be handled from within the useStyles method to not use the Hook. See line 14

Reply:
After the simplification of the joint key review step page the alternating colors looked like they were being used to indicate different types of guardians.  The colored guardians may well fit into a different page, so I didn't remove the functionality.  Good point about moving the logic to useSyles, fixed.